shell/shared/demo.js

```diff
@@ -162,3 +176,34 @@ Router.map(function () {
     }
   });
 });
+
+Router.map(function () {
+  this.route("appdemo", {
+    path: "/appdemo/:packageId",
+    waitOn: function () {
+      return Meteor.subscribe("packageInfo", this.params.packageId);
+    },
+    onAfterAction: function() {
+      /* Stash the packageId into the Meteor session, so that, after
+	 creating a new demo user, we know which app to start a grain
+	 for.
+      */
+      Session.set("packageId", this.params.packageId);
```

Comment:
I can use `packageId` in the template that way, but I couldn't figure out how to use it from Javascript code except by adding it to the session.